### PR TITLE
Add new entries to remote-config

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

It was time for some new entries.

### CODE REVIEW

File: remote-config.jsonc [hunk 1]
```diff
@@ -93,6 +93,36 @@
                 },
 				{
                     "message": "MacStadium supports DDEV with an M1 Mac Mini that runs macOS tests"
+                },
+                {
+                    "message": "You can turn off the cell borders in `ddev describe` and `ddev list` with `ddev config global --simple-formatting`"
```
Comment (mattstein) [inline remote-config.jsonc]: Run commands, upgrade to a complete sentence:

```suggestion
                    "message": "You can turn off the cell borders in `ddev describe` and `ddev list` by running `ddev config global --simple-formatting`."
```

File: remote-config.jsonc [hunk 2]
```diff
@@ -93,6 +93,36 @@
                 },
 				{
                     "message": "MacStadium supports DDEV with an M1 Mac Mini that runs macOS tests"
```
Comment (mattstein) [inline remote-config.jsonc]: I’m adding scope creep, but these might as well follow the same format and be complete sentences with punctuation: 

```suggestion
                    "message": "MacStadium supports DDEV with an M1 Mac Mini that runs macOS tests."
```

File: remote-config.jsonc [hunk 3]
```diff
@@ -93,6 +93,36 @@
...
+                },
+                {
+                    "message": "You can turn off the cell borders in `ddev describe` and `ddev list` with `ddev config global --simple-formatting`"
+                },
+                {
+                    "message": "You can use a more lively `ddev list` and `ddev describe` output with `ddev config global --table-style=bright`. Try it out!"
```
Comment (mattstein) [inline remote-config.jsonc]: Run commands, drop “out” to save characters with no change in meaning:

```suggestion
                    "message": "You can use a more lively `ddev list` and `ddev describe` output by running `ddev config global --table-style=bright`. Try it!"
```

File: remote-config.jsonc [hunk 4]
```diff
@@ -93,6 +93,36 @@
...
+                },
+                {
+                    "message": "DDEV's fiscal entity is now the DDEV Foundation. http://ddev.com/foundation. We have a real focus on the long-term sustainability of the project, not just depending on Randy."
```
Comment (mattstein) [inline remote-config.jsonc]: I’d let the message read in its entirety before including the link as a way to go and learn more. (And use an HTTPS URL.)

```suggestion
                    "message": "DDEV's fiscal entity is now the DDEV Foundation. We have a real focus on the long-term sustainability of the project, not just depending on Randy. https://ddev.com/foundation"
```

File: remote-config.jsonc [hunk 5]
```diff
@@ -93,6 +93,36 @@
...
+                },
+                {
+                    "message": "Tag1 Consulting has become a major sponsor of DDEV with $1K/month! Stop by and thank them, https://tag1.com."
```
Comment (mattstein) [inline remote-config.jsonc]: I’d make the URL part of the sentence, instead of having the comma add an unnecessary pause without any supporting language:

```suggestion
                    "message": "Tag1 Consulting has become a major sponsor of DDEV with $1K/month! Stop by and thank them at https://tag1.com."
```

File: remote-config.jsonc [hunk 6]
```diff
@@ -93,6 +93,36 @@
...
+                },
+                {
+                    "message": "DDEV now has PHP 8.3 in latest releases. We hope to have all extensions by the time of 8.3 release."
```
Comment (mattstein) [inline remote-config.jsonc]: Reword to reduce a slight caveperson sound and improve clarity:

```suggestion
                    "message": "DDEV’s latest releases include PHP 8.3. We hope to have all extensions by the time 8.3 reaches general availability later this year."
```

File: remote-config.jsonc [hunk 7]
```diff
@@ -93,6 +93,36 @@
...
+                },
+                {
+                    "message": "Lots of database browsers work with DDEV, including TablePlus, Sequel Pro, and DBeaver. https://ddev.readthedocs.io/en/latest/users/usage/database-management/#database-guis"
+                },
+                {
+                    "message": "View officially supported add-ons with `ddev get --list` or ALL of them with `ddev get --list --all`"
```
Comment (mattstein) [inline remote-config.jsonc]: Hyphenate officially-supported, run commands, reword to distinguish between officially-supported and third-party where “all” is more vague:

```suggestion
                    "message": "View officially-supported add-ons by running `ddev get --list`, or include third-party add-ons with `ddev get --list --all`"
```

File: remote-config.jsonc [hunk 8]
```diff
@@ -93,6 +93,36 @@
...
+                },
+                {
+                    "message": "vscode users: Have you tried the new DDEV vscode extension? https://marketplace.visualstudio.com/items?itemName=biati.ddev-manager"
```
Comment (mattstein) [inline remote-config.jsonc]: “VS Code” for stylistic consistency:

```suggestion
                    "message": "VS Code users: have you tried the new DDEV VS Code extension? https://marketplace.visualstudio.com/items?itemName=biati.ddev-manager"
```

File: remote-config.jsonc [hunk 9]
```diff
@@ -93,6 +93,36 @@
...
+                },
+                {
+                    "message": "PhpStorm users: You'll love the DDEV Integration Plugin for PhpStorm, https://plugins.jetbrains.com/plugin/18813-ddev-integration"
```
Comment (mattstein) [inline remote-config.jsonc]: I very subjectively feel like it’s worth ending the sentence with a bang instead of rolling the link over a comma:

```suggestion
                    "message": "PhpStorm users: you'll love the DDEV Integration Plugin for PhpStorm! https://plugins.jetbrains.com/plugin/18813-ddev-integration"
```

File: remote-config.jsonc [hunk 10]
```diff
@@ -93,6 +93,36 @@
...
+                },
+                {
+                    "message": "Did you know you can set environment variables with the .ddev/.env file?"
```
Comment (mattstein) [inline remote-config.jsonc]: If commands are in backticks (like the docs), this file reference should be too:

```suggestion
                    "message": "Did you know you can set environment variables with the `.ddev/.env` file?"
```

**Review by mattstein — CHANGES_REQUESTED** [review]
Nitpicks nitpicked!

File: remote-config.jsonc (hunk 8 above)
Comment (rfay) [reply remote-config.jsonc]: Interesting, I thought it was always just vscode.

**Comment by rfay** [comment]
Is THAT all you've got? Surely you can do more!
